seatunnel-connectors-v2/connector-elasticsearch/src/main/java/org/apache/seatunnel/connectors/seatunnel/elasticsearch/config/ElasticsearchBaseOptions.java

```diff
@@ -87,4 +87,30 @@ public class ElasticsearchBaseOptions implements Serializable {
                     .stringType()
                     .noDefaultValue()
                     .withDescription("The key password for the trust store specified");
+
+    // Authentication configuration options
+    public static final Option<String> AUTH_TYPE =
+            Options.key("auth_type")
+                    .stringType()
```

Comment:
We can use enum type for `auth_type`.
```suggestion
                    .enumType()
```